crate/server/src/core/cover_crypt/update_policy.rs

```diff
@@ -138,6 +170,123 @@ fn rotate_policy(
     Ok(policy)
 }
 
+/// Removes old rotations from the policy of the given Master Private Key
```

Comment:
Same as a previous comment, I would rather like we make it clear that rotations associate values to attributes, thus the comment could be "removes the old attribute values from the policy".